src/Sentry/SpanTracer.cs

```diff
@@ -190,5 +191,13 @@ internal set
             _origin = value;
         }
     }
-    private string? _origin;
+
+    /// <summary>
+    /// Automatically finishes the span at the end of a <c>using</c> block. This is a convenience method only. Disposing
+    /// is not required.
+    /// </summary>
```

Comment:
suggestion: shall we update this comment to be similar to the comment on `TransactionTracer.Dispose`?

```suggestion
    /// <summary>
    /// <para>
    /// Automatically finishes the span with a status of <see cref="SpanStatus.Ok" /> at the end of a
    /// <c>using</c> block, if it has not already been finished.
    /// </para>
    /// <para>
    /// This is the equivalent of calling <see cref="Finish()" /> when the transaction passes out of scope.
    /// </para>
    /// </summary>
    /// <remarks>This is a convenience method only. Disposing is not required.</remarks>
```